octobot_trading/enums.py

```diff
@@ -328,12 +357,14 @@ class ExchangeConstantsOrderColumns(enum.Enum):
     SYMBOL = "symbol"
     QUANTITY_CURRENCY = "quantity_currency"
     TYPE = "type"
+    OCTOBOT_ORDER_TYPE = "octobot_order_type"
```

Comment:
What will it be used for?

Reply:
We parse two different order types. For market order, the type would be "market" TradeOrderType and octobot oder type "buy_market" TraderOrderType
It's kind of temporary, we use both now and later I want to change all references to TraderOrderType

Reply:
I don't understand why we can't still use only `type`

Reply:
in the Order class we use self.exchange_order_type and self.order_type. 
I moved the logic from Order to OrderParser. 
We can remove one of them once we refactor Orders and only require one of them